JitsiConference.js

```diff
@@ -2854,8 +2856,16 @@ JitsiConference.prototype._onIceConnectionFailed = function(session) {
             reasonDescription: 'ICE FAILED'
         });
     } else if (session && this.jvbJingleSession === session) {
+        // Implement an exponential backoff timer for ICE restarts.
+        const delay = Math.min(1000 * Math.pow(2, this._iceRestarts), 60000);
+        const jitter = Math.random() * 100; // Adding jitter to avoid synchronized retries
+        const totalDelay = delay + jitter;
```

Comment:
In other places we use getJitterDelay, can we use it here?

Reply:
Sure, didn't know we had such a thing. Thanks for pointing it out.